pkgs/by-name/su/surrealist/package.nix

```
@@ -23,7 +23,24 @@
 
 let
 
-  esbuild-20-2 = let version = "0.20.2";
+  cargo-tauri-2 = let
```

```suggestion
  cargo-tauri_2 = let
```
versions are usually suffixed with underscore